:bug: Dependabot PR titles are not parsing

We get 

```
Traceback (most recent call last):
  File "/home/runner/work/_actions/pyiron/actions/main/update-env-files/../.support/update_environment.py", line 116, in <module>
    raise ValueError(f"Title of a dependabot PR 'Bump <package> from <version> to <version>' expected, "
ValueError: Title of a dependabot PR 'Bump <package> from <version> to <version>' expected, but got Bump pyiron-base from 0.6.8 to 0.6.9 .ci_support/environment.yml /home/runner/work/_actions/pyiron/actions/main/update-env-files/../.support/pypi_vs_conda_names.json
Error: Process completed with exit code 1.
```

Which I guess is a result of #56.

I'm working on it right now, but @niklassiemer FYI.

--- a/.support/update_environment.py
+++ b/.support/update_environment.py
@@ -72,7 +72,7 @@ def ends_in_yml(s):
 
 
 def pattern_is_bump_from_to(list_):
-    if len(list_) < 10:
+    if len(list_) < 9:
         return False
     return all(
         [
@@ -85,7 +85,7 @@ def pattern_is_bump_from_to(list_):
 
 
 def pattern_is_update_requirement_from_to(list_):
-    if len(list_) < 11:
+    if len(list_) < 10:
         return False
     return all(
         [

